### DISCUSSION TITLE
Few suggestions for generated rss feeds

### DISCUSSION THREAD

**Discussion by followmyrandomness** [GetPublii/Publii#1765] [Ideas]
Apologies if this has already been mentioned:

Upon reviewing the RSS XML/JSON generated by Publii, I noticed that hidden tags are still being included. I was hoping that hidden tags would remain completely hidden, as my understanding is that they are primarily intended to simplify the publisher's workflow in cases where minor tweaks to the website's logic are needed.

Additionally, when I mark certain posts as 'excluded from the homepage', these posts still appear in the main homepage's RSS feed. I was expecting them not to be included. Ideally, if I mark a post as 'excluded from the homepage' without fully hiding it, I would prefer that it remain available only in the relevant category/tag-specific feed. For instance, if each tag had its own RSS feed, I would expect those posts to appear there rather than in the main RSS feed.

Thank you.

**Comment by bjazmoore** [comment]
You might click on the Issues link in the top links bar of GitHub and create an issue for these problems.

**Comment by dziudek** [comment]
@followmyrandomness 

> Upon reviewing the RSS XML/JSON generated by Publii, I noticed that hidden tags are still being included. I was hoping that hidden tags would remain completely hidden, as my understanding is that they are primarily intended to simplify the publisher's workflow in cases where minor tweaks to the website's logic are needed.

What do you mean by hidden tags?

> Additionally, when I mark certain posts as 'excluded from the homepage', these posts still appear in the main homepage's RSS feed. I was expecting them not to be included. Ideally, if I mark a post as 'excluded from the homepage' without fully hiding it, I would prefer that it remain available only in the relevant category/tag-specific feed. For instance, if each tag had its own RSS feed, I would expect those posts to appear there rather than in the main RSS feed.

You're right - posts excluded from the homepage should be hidden on the RSS feed too - I will check this and fix in v.0.46.2 planned for the next week

**Reply by followmyrandomness** [reply]
Thanks for the reply!

Regarding hidden tags, I'll provide an example:

I’ve set a tag '0_eng' that I have marked as hidden; this tag is only for my own purposes for posts in English. I also have a tag '0_general_title_date_only' which helps me modify the theme so that for posts with this tag, only the post's date is displayed as the title.

When I look into the feed.xml of the generated output, I see the following:

```
<category term="wificalling"/>
<category term="volte"/>
<category term="thoughts"/>
<category term="tech"/>
<category term="spusu"/>
<category term="observations"/>
<category term="mobilenetworks"/>
<category term="mobilenetwork"/>
<category term="mobile"/>
<category term="giffgaff"/>
<category term="callforwarding"/>
<category term="0_general_title_date_only"/>
<category term="0_eng"/>
```


These are the tags for one particular post in the RSS feed. It’s important to note that not only the "normal" tags are shown; the hidden ones are as well. As I understand it (though I may be wrong), the purpose of hidden tags is to assist the website's developers in enhancing the site's logic in some way.

**Reply by dziudek** [reply]
@followmyrandomness - ok, now all is clear :) Sorry but I thought that by `hidden tags` you mean HTML tags :)

Regarding hidden tags for posts - I agree - it should be excluded from RSS - it will be also added as a fix in v.0.46.2
